pkg/cluster/internal/create/actions/kubeadmjoin/join.go

```diff
@@ -77,7 +77,7 @@ func joinSecondaryControlPlanes(
 	allNodes []nodes.Node,
 	secondaryControlPlanes []nodes.Node,
 ) error {
-	ctx.Status.Start("Joining control-plane nodes 🎮")
+	ctx.Status.Start("Joining more control plane nodes 🎮")
```

Comment:
i personally preferred the old message.

Reply:
> i personally preferred the old message.

+1  :+1:

Reply:
FWIW a next step soonish is to redo the output:
- logging should only be used for debug, user facing messages of any sort should not be logging
- status may be redone to change how we do spinning (or remove spinners and do dots or... have some experiments, haven't refactored them into kind yet)
- the emoji need more thought to pick the right ones 🤔